### ISSUE TITLE
Escaping broken for passing multiple arguments

### ISSUE THREAD

**Issue by fadookie** [bitrise-steplib/steps-npm#9]
After upgrading from 0.9.1 to 1.0.0, all our builds are failing during `npm install` operation.

Steps to reproduce:
1. Make a build with an npm command like so: `install --loglevel warn`

Expected:
`npm install --loglevel warn` is invoked

Actual:
`npm "install --loglevel warn"` is invoked, resulting in an error:

```
+------------------------------------------------------------------------------+

| (4) npm install dev-utils                                                    |
+------------------------------------------------------------------------------+
| id: npm                                                                      |
| version: 1.0.0                                                               |
| collection: https://github.com/bitrise-io/bitrise-steplib.git                |
| toolkit: go                                                                  |
| time: 2018-11-21T08:32:02-08:00                                              |
+------------------------------------------------------------------------------+
|                                                                              |
Config:
- Workdir: /Users/vagrant/dev-utils
- Command: install --loglevel warn
- NpmVersion: 
Autodetecting npm version
Checking package.json for npm version
No npm version found in package.json
Locating system installed npm
npm found at /Users/vagrant/.nvm/versions/node/v8.13.0/bin/npm
$ npm "--version"
6.4.1
Running user provided command
$ npm "install --loglevel warn"
Error running command install --loglevel warn: error running npm command: exit status 1
|                                                                              |
+---+---------------------------------------------------------------+----------+
| x | npm install dev-utils (exit code: 1)                          | 4.07 sec |
+---+---------------------------------------------------------------+----------+
| Issue tracker: https://github.com/bitrise-steplib/steps-npm/issues           |
| Source: https://github.com/bitrise-steplib/steps-npm                         |
+---+---------------------------------------------------------------+----------+
```

**Comment by jkdoolan** [comment]
+1

**Comment by lszucs** [comment]
Hi @fadookie,

We have just published version 1.0.1, which fixes issues with npm commands made up of several components.

For more info, see: https://github.com/bitrise-steplib/steps-npm/releases/tag/1.0.1

For now, I'll close this issue, but if you need more info/help feel free to open a new issue!
